src/um/commoncontrols.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright © 2015-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use ctypes::{c_int, c_void};
+use shared::guiddef::{REFCLSID, REFIID};
+use shared::minwindef::{BOOL, DWORD, LRESULT, UINT};
+use shared::windef::{COLORREF, HBITMAP, HICON, HWND, POINT, RECT};
+use um::commctrl::{IMAGEINFO, IMAGELISTDRAWPARAMS};
+use um::unknwnbase::{IUnknown, IUnknownVtbl};
+use um::winnt::HRESULT;
+extern "system" {
+    pub fn ImageList_CoCreateInstance(
+        rclsid: REFCLSID,
+        punkOuter: *const IUnknown,
+        riid: REFIID,
+        ppv: *mut *mut c_void,
+    ) -> HRESULT;
+}
+pub const ILIF_ALPHA: DWORD = 0x00000001;
+pub const ILIF_LOWQUALITY: DWORD = 0x00000001;
+pub const ILDRF_IMAGELOWQUALITY: LRESULT = 0x00000001;
+pub const ILDRF_OVERLAYLOWQUALITY: LRESULT = 0x00000010;
+RIDL!{#[uuid(0x46EB5926, 0x582E, 0x4017, 0x9F, 0xDF, 0xE8, 0x99, 0x8D, 0xAA, 0x09, 0x50)]
```

Comment:
The uuid attribute should be lowercase hex: https://github.com/retep998/winapi-rs/blob/dev/CONTRIBUTING.md#com-interfaces